modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc

```diff
@@ -152,6 +152,6 @@ Examples are:
 
 == Long Running Uploads
 
-For very long running uploads (those lasting longer than 1 hr) to public folders (and chunking is not in effect), 'filelocking.ttl' should be set to a significantly large value. 
-If this is not set appropriately, then large file uploads will likely fail.
-A simple way to calculate the correct value is by using the formula: `time in seconds = (maximum file size / slowest assumed connection)`.
+For very long running uploads (those lasting longer than 1 hr) to public folders (and chunking is not in effect), 'filelocking.ttl' should be set to a significantly larger value. 
+If this is not set appropriately, then large file uploads will fail with a file locking error, because the Redis garbage collection will delete the initially acquired file lock after 1 hour by default.
+To estimate a good value use this formula: `time in seconds = (maximum file size / slowest assumed connection)`.
```

Comment:
@enbrnz, pls excuse my lack of understanding here, but what do you may by "_slowest assumed connection_"?

Reply:
didn't I respond to this exact question in the last issue /  PR?

Reply:
After talking to Erik, we need to specify that you need to take your UPLOAD speed and half it. then put that number in the formula. So for example if you have 50 000 down and 10 000 up you need to half the 10 000 and calculate half that if I understand correctly.

Reply:
Slowest assumed connection: Somebody has to provide ownCloud reliably to users in remote areas with slow connections, e.g. 8MBit/s DSL connection where the 8MBit/s stands for the download speed. This means that the user probably has contractually an upload speed of about 1MBit/s. If the speed is flaky during times of high load, probably more half that.

Reply:
Thanks for the clarification @voroyam & @enbrnz.